_commands/basics/man.md

```diff
@@ -3,22 +3,31 @@
 
 man
 --
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+`man` provides *manual pages* for terminal commands
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ man git

```

Comment:
nit: add another comment that says something like
```
$ man git
# displays git man page
q # q to quit
```
Right now it seems that the command `man git` doesn't do anything.